buildenv/jenkins/docker-slaves/x86/centos6.9/Dockerfile

```diff
@@ -111,24 +111,15 @@ RUN wget -O - http://cpanmin.us | perl - --self-upgrade \
   && cpanm Text::CSV \
   && cpanm JSON
 
-# install GCC-4.8 and GCC-7.3
-RUN cd /etc/yum.repos.d \
-  && wget http://people.centos.org/tru/devtools-2/devtools-2.repo -O /etc/yum.repos.d/devtools-2.repo \
-  && yum -y update \
-  && yum -y install devtoolset-2-gcc devtoolset-2-binutils \
-  && yum -y install devtoolset-2-gcc-c++ devtoolset-2-gcc-gfortran \
-  && yum clean all \
-  && scl enable devtoolset-2 bash \
-  && mv /opt/rh/devtoolset-2/root/usr/bin/gcc /opt/rh/devtoolset-2/root/usr/bin/gcc-4.8 \
-  && mv /opt/rh/devtoolset-2/root/usr/bin/g++ /opt/rh/devtoolset-2/root/usr/bin/g++-4.8 \
-  && ln -s /opt/rh/devtoolset-2/root/usr/bin/gcc-4.8 /usr/bin/gcc \
-  && ln -s /opt/rh/devtoolset-2/root/usr/bin/g++-4.8 /usr/bin/g++ \
-  && ln -s /opt/rh/devtoolset-2/root/usr/bin/gcc-4.8 /usr/bin/cc \
-  && yum -y update \
+# Install GCC-7.3
+RUN yum -y update \
   && yum -y install centos-release-scl \
   && yum -y install devtoolset-7-gcc devtoolset-7-binutils \
   && yum -y install devtoolset-7-gcc-c++ devtoolset-2-gcc-gfortran \
-  && yum clean all
+  && yum clean all \
+  && ln -s /opt/rh/devtoolset-7/root/usr/bin/gcc /usr/bin/cc \
+  && ln -s /opt/rh/devtoolset-7/root/usr/bin/gcc /usr/bin/gcc \
+  && ln -s /opt/rh/devtoolset-7/root/usr/bin/g++ /usr/bin/g++ 
```

Comment:
Is it necessary to symlink default gcc? I.e. does something need a default gcc?

Reply:
git requires cc and Protobuf requires gcc.

Reply:
ok good enough. I doubt compiler versions matter too much for those dependencies. Unless they fail outright.

Reply:
I'm not surprised this didn't cause a problem.  Running gcc out of the devtoolset requires running a script to setup the environment and libpaths for newer libraries.  If this generates a docker container that compiles openj9, I guess we leave it be though.